redhat-operators/amq-streams/amq-streams.package.yaml

```diff
@@ -1,5 +1,5 @@
-#! package-manifest: deploy/chart/catalog_resources/rh-operators/amq-streams.v1.0.0.clusterserviceversion.yaml
+#! package-manifest: ./deploy/chart/catalog_resources/rh-operators/amq-streams.v1.0.0-clusterserviceversion
 packageName: amq-streams
 channels:
-- name: preview
-  currentCSV: amqstreams.v1.0.0.beta
+- name: final
+  currentCSV: amqstreams.v1.0.0
```

Comment:
Do we need to consider the scenario where an openshift user is running `amqstreams.v1.0.0.beta` in an environment and wants to upgrade to `amqstreams.v1.0.0`?

Reply:
I thought so but wanted to make sure. 

Yes, OLM handles upgrade ( automatic or via manual approval ) to the new version. The way to go about is to treat the `csv` as an immutable object once it's in GA, and then adding a new `csv` for subsequent release(s). 

We will write up documentation on how to extend `csv` for upgrade(s) to help your team guide through it.